lineman/app/app.coffee

```diff
@@ -56,6 +56,9 @@ angular.module('loomioApp').controller 'ApplicationController', ($scope, $filter
 
   $scope.keyDown = (event) -> KeyEventService.broadcast event
 
+  if $location.search().feedback_given?
```

Comment:
NB: this will cause a merge conflict with the new BootService in the 'new groups start on angular' PR.